src/testing/Test.java

```diff
@@ -1,10 +1,8 @@
 package testing;
 
 public class Test {
-	boolean  a = true;
+	private static int a;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "a" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1068)